lib/SyTest/Synapse.pm

```diff
@@ -115,7 +115,8 @@ sub start
    }
 
    if( defined $db_type ) {
-      $self->${\"clear_db_$db_type"}( %db_args );
+      my $clear_meth = "clear_db_${db_type}";
+      $self->${\$clear_meth}( %db_args );
```

Comment:
Thanks. TBH the main reason I pulled it out was because my editor was confused by `\"`.